src/custom/components/CurrencyInputPanel/index.tsx

```diff
@@ -1,23 +1,97 @@
 // import { darken } from 'polished'
 import React from 'react'
-import styled, { css } from 'styled-components'
+import styled, { css } from 'styled-components/macro'
 import { darken } from 'polished'
 import useLoadingWithTimeout from 'hooks/useLoadingWithTimeout'
 import { useIsQuoteRefreshing } from 'state/price/hooks'
+import { FeeInformationTooltipWrapper } from 'components/swap/FeeInformationTooltip'
+import CurrencySearchModalUni from 'components/SearchModal/CurrencySearchModal'
+import { ReactComponent as DropDown } from 'assets/images/dropdown.svg'
 
-import CurrencyInputPanelMod, {
-  CurrencyInputPanelProps,
-  CurrencySelect as CurrencySelectMod,
-  InputRow,
-  InputPanel,
-  LabelRow,
-  Container,
-} from './CurrencyInputPanelMod'
+import CurrencyInputPanelMod, { CurrencyInputPanelProps } from './CurrencyInputPanelMod'
 import { RowBetween } from 'components/Row'
 
 import { StyledLogo } from 'components/CurrencyLogo'
 import { LONG_LOAD_THRESHOLD } from 'constants/index'
 
+export const CurrencySearchModal = styled(CurrencySearchModalUni)`
```

Comment:
Added all the styled components from the Mod file into the `index` file.

Reply:
i think you should revert this unless it's a new component. such as this one, this can go to the index 👍

Reply:
Reverted now.